python/lsst/qserv/tests/benchmark.py

```diff
@@ -177,6 +174,13 @@ def _parseFile(self, qF, withQserv):
             elif withQserv and line.startswith("-- withQserv"):
                 # strip the "-- withQserv" text
                 qText.append(line[13:])
+            elif line.endswith("-- noQserv"):
+                if withQserv:
+                    # skip this line
+                    pass
+                else:
+                    # strip the "-- noQserv" text
+                    qText.append(line[:-10])
```

Comment:
Yes, test passes. I think that "-- noQserv" should start the line, like other pragmas